package.json

```diff
@@ -78,20 +78,20 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.3.4",
-    "@bbc/gel-foundations": "^0.2.4",
+    "@bbc/gel-foundations": "^0.3.0",
```

Comment:
Last time I did this James wanted a list of all possible changes to these packages like https://github.com/bbc/simorgh/pull/1394. Its made by comparing the versions and manually copying the changes in their changelog